docs/my-app/main.js

```diff
@@ -8,7 +8,7 @@ define(['angular'
 , 'portal/notifications/route'
 , 'portal/features/route'
 , 'portal/about/route'
-, 'my-app/home/route',
+, 'my-app/home/route'
```

Comment:
Nope -- there was an extra comma before (one after `my-app/home/route` and one before `my-app/home/controllers`)